contrib/systemd/mako.service.in

```diff
@@ -3,11 +3,16 @@ Description=Lightweight Wayland notification daemon
 Documentation=man:mako(1)
 PartOf=graphical-session.target
 After=graphical-session.target
+# ConditionEnvironment requires systemd v247 to work correctly
+ConditionEnvironment=WAYLAND_DISPLAY
 
 [Service]
 Type=dbus
 BusName=org.freedesktop.Notifications
-ExecCondition=/bin/sh -c '[ -n "$WAYLAND_DISPLAY" ]'
+
```

Comment:
I'd remove old code rather than commenting out -- if anyone cares, it's still in the git logs.

Reply:
OK, I removed the old code with an amended commit.

Reply:
You should probably remove the extra newline you added as well.